docker/build-tools/Dockerfile

```diff
@@ -80,7 +80,7 @@ RUN mkdir -p ${OUTDIR}/usr/local
 
 # Update distro and install dependencies
 # hadolint ignore=DL3008
-RUN apt-get update && apt-get -y dist-upgrade && apt-get install -y --no-install-recommends \
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Comment:
Was dist-upgrade causing hadolint issues?

Reply:
Yes, and I agree with the rationale - this is provided by the base image updating. And something about it causing things to get out of sync that I didn't fully understand